shared/resources/build.gradle.kts

```diff
@@ -30,11 +34,11 @@ kotlin {
         commonMain {
             dependencies {
                 api(libs.moko.resources)
+                implementation(libs.jetbrains.compose.runtime)
```

Comment:
Why JB Compose runtime in commonMain instead of Jetpack Compose runtime in androidMain?

Reply:
It's because compose compiler moved to kotlin repo since 2.0.0